test/eslint.js

```diff
@@ -64,5 +64,6 @@ test('should fail with enabled throwError option', t => {
         t.fail('should throw error');
     }).catch(err => {
         t.is(err.toString(), 'ESLintError: Warnings or errors were found');
+        t.is(err.stack, null);
```

Comment:
Tried with gulp and in this case I got stack trace of gulp itself. I think the best way is something like
```js
err.stack = err.toString();
```